lib/hammer_cli_katello/repository.rb

```diff
@@ -33,7 +33,8 @@ class InfoCommand < HammerCLIKatello::InfoCommand
         field :url, _("URL")
         field :_publish_via_http, _("Publish Via HTTP")
         field :full_path, _("Published At")
-
+        field :docker_upstream_name, _("Upstream Repository Name"),
+                                      Fields::Field, :hide_blank => true
```

Comment:
well also (more so in katello) we have a lot of cops simply turned of instead of changed their options. with rubocop `0.28.0`, a lot of cops are configurable.